src/test/java/hw/four/AbstractSettingClass.java

```diff
@@ -0,0 +1,36 @@
+package hw.four;
+
+import driverutils.ChromeDriverManager;
+import driverutils.SingletonForProperties;
+import hw.four.steps.ActionStep;
+import hw.four.steps.AssertionStep;
+import org.openqa.selenium.WebDriver;
+import org.testng.ITestContext;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public abstract class AbstractSettingClass {
+
+    public SingletonForProperties singletonForProperties = SingletonForProperties.getInstance();
+    public WebDriver driver;
+
+    ActionStep actionStep;
+    AssertionStep assertionStep;
```

Comment:
These class fields can be protected not public

Reply:
changed